elasticsearch/tests/elasticsearch_test.py

```diff
@@ -899,3 +900,80 @@ def test_keystore_volumes():
                  }]
              }
            } in s['volumes']
+def test_pod_security_policy():
+    ## Make sure the default config is not creating any resources
+    config = '''
```

Comment:
Since this is supposed to be testing the defaults we should leave the config set to `config = ''` to make sure that the test is still valid if the defaults are ever changed in a future major release.

Reply:
And to guarantee that we are actually testing the defaults and that there aren't any copy paste mistakes.

Reply:
Done in [e7f76dc](https://github.com/elastic/helm-charts/pull/265/commits/e7f76dce3a69c451ff72830cbef57e62631663e1).